src/simple/Phone.js

```diff
@@ -221,9 +221,14 @@ class Phone extends Emitter {
     });
   }
 
-  checkSfu() {
-    const hasSfu = this.sipLine.options.some(option =>
+  getSfu() {
+    return this.sipLine && this.sipLine.options.some(option =>
       option[0] === 'max_audio_streams' || option[0] === 'max_video_streams');
```

Comment:
Not sure about this check, we need to have more than one max_video_streams and more than zero for max_audio_streams.

Reply:
Noted. 

Note: closing this PR, we'll be handling the SFU check directly in WDA for now